mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixSwagger.Mixfile do
   use Mix.Project
 
-  @version "0.6.4"
+  @version "0.6.5"
```

Comment:
The reason I say that is because we're not adding functionality here. There is no added functionality. It's simply fixing a broken feature. Nobody should be using pre-release software when the release is available.